packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -225,51 +225,50 @@ class SlideTransition extends AnimatedWidget {
   }
 }
 
-/// Animates the scale of a transformed widget.
+/// Signature for the callback to [MatrixTransition.onTransform].
 ///
-/// Here's an illustration of the [ScaleTransition] widget, with it's [alignment]
-/// animated by a [CurvedAnimation] set to [Curves.fastOutSlowIn]:
-/// {@animation 300 378 https://flutter.github.io/assets-for-api-docs/assets/widgets/scale_transition.mp4}
-///
-/// {@tool dartpad}
-/// The following code implements the [ScaleTransition] as seen in the video
-/// above:
+/// Computes a [Matrix4] to be used in the [MatrixTransition] transformed widget
+/// from the [MatrixTransition.animation] value.
+typedef TransformCallback = Matrix4 Function(double);
+
+/// Animates the [Matrix4] of a transformed widget.
 ///
-/// ** See code in examples/api/lib/widgets/transitions/scale_transition.0.dart **
-/// {@end-tool}
+/// The [onTransform] callback computes a [Matrix4] from the animated value.
 ///
```

Comment:
Bonus points if you add some sample code to demonstrate how this is used, similar to what is showcased for the ScaleTransition in `examples/api/lib/widgets/transitions/scale_transition.0.dart`.

(This is strictly optional, though)

Reply:
I'd be happy to. I'd rather do it in a follow up PR.